src/FarmCraft.Core/Services/Messaging/Consumer/ServiceBusConsumer.cs

```diff
@@ -1,20 +1,20 @@
 ﻿using Azure.Messaging.ServiceBus;
 using FarmCraft.Core.Messaging;
+using FarmCraft.Core.Services.Messaging.Handler;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Newtonsoft.Json;
 using System.Collections.Concurrent;
 
-namespace FarmCraft.Core.Services.Messaging
+namespace FarmCraft.Core.Services.Messaging.Consumer
 {
     /// <summary>
     /// A Message Consumer for interacting with Azure Service Bus
     /// </summary>
     public class ServiceBusConsumer : IMessageConsumer, IDisposable
     {
         private readonly ILogger _logger;
-        private static ServiceBusClient _client;
-        private static ServiceBusProcessor _processor;
+        private static ServiceBusProcessor? _processor;
```

Comment:
This can't be static if there will be more than one consumer. Test to make sure this works correctly if registering multiple instances as singletons, or maybe rethink this a bit.